kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue

```diff
@@ -1,30 +1,26 @@
 <template>
 
-  <CoreBase
-    :immersivePage="false"
+  <CoachAppBarPage
     :authorized="userIsAuthorized"
     authorizedRole="adminOrCoach"
     :showSubNav="true"
   >
-    <template #sub-nav>
-      <TopNavbar />
-    </template>
 
     <KPageContainer>
       <PlanHeader />
 
       <div class="filter-and-button">
         <!-- Hidden temporarily per https://github.com/learningequality/kolibri/issues/6174
-        <KSelect
-          v-model="statusSelected"
-          :label="coreString('showAction')"
-          :options="statusOptions"
-          :inline="true"
-        />
-        -->
+          <KSelect
+            v-model="statusSelected"
+            :label="coreString('showAction')"
+            :options="statusOptions"
+            :inline="true"
+          />
+          -->
         <!-- Remove this div - it makes sure the [NEW LESSON] button stays right-aligned
-            while the above <KSelect> is hidden
-        -->
+              while the above <KSelect> is hidden
+          -->
```

Comment:
I created #9886 so that we can get this uncommented and working again; I'm glad that it got linted or else we may not have noticed they were missing again until it was too late :sweat_smile:

Reply:
I am more glad i didn't get rid of it. :)

Reply:
Issue will be resolved with #9886. Thanks @nucleogenesis